.gitignore

```diff
@@ -49,6 +49,9 @@ lib/libesp32/berry/berry
 *.bak
 *.code-workspace
 
+## IntelliJ ######
+.idea
```

Comment:
I used [CLion](https://www.jetbrains.com/clion/), so I've completed `.gitignore` to support this IDE.

Reply:
IDE specific ignores should be part of your personal git-ignore configuration, but not for each project.

Reply:
My philosophy is the following: if it may help others, I think I should add it. Else, you may discourage developers using CLion, which, by the way, is used by a lot of people. Moreover, this gitignore, like many others, includes vscode exclusion. So the final question is: do you prefer to add one line that could help and include more developers, or do you prefer strictness?

Depending on the vision of the codeowner, I'll keep it or remove it from the PR.

Reply:
There is no harm in adding this line in `.gitignore` if it makes your life easier